stories/side-navigation/side-navigation.stories.js

```diff
@@ -1254,3 +1254,85 @@ nestedListWithoutLinks.parameters = {
         disable: true
     }
 };
+
+export const nestedListPopover = () => `<div class="fd-popover fd-popover--right" style="margin-bottom: 200px; margin-right: 200px; margin-left: 200px;">
+    <div class="fd-popover__control">
+        <ul class="fd-nested-list fd-nested-list--compact">
+            <li class="fd-nested-list__item">
+                <a class="fd-nested-list__link"href="#/">
+                    <i role="presentation" class="fd-nested-list__icon sap-icon--home"></i>
+                </a>
+            </li>
+        </ul>
+    </div>
+    <div class="fd-popover__popper" aria-hidden="false" id="popoverA11">
+        <ul class="fd-nested-list fd-nested-list--popover fd-nested-list--compact">
+            <li class="fd-nested-list__group-header fd-nested-list__group-header--popover">
+                Group Header 1
+            </li>
+            <li class="fd-nested-list__item fd-nested-list__item--popover">
+                <a class="fd-nested-list__link"href="#/">
```

Comment:
```suggestion
                <a class="fd-nested-list__link" href="#/">
```
and in other `<a>` tags in this example

Reply:
other places in documentation has this format. so i did not change that.
i have fixed it now.

Reply:
do storyshots need updating after this change?